### ISSUE TITLE
Xcode 14 Build Warning

### ISSUE THREAD

**Issue by LowAmmo** [mattgallagher/CwlCatchException#7]
Seeing this build warning in CwlCatchException line 52 - 

     public struct ExceptionError: CustomNSError {
	     public let exception: NSException
	     public let domain = "com.cocoawithlove.catch-exception"
	     public let errorUserInfo: [String: Any]

Xcode's warning - "Stored property 'exception' of 'Sendable'-conforming struct 'ExceptionError' has non-sendable type 'NSException'"

Not sure how to address this warning... Unless it's easy to make NSException conform to "Sendable"?

**Comment by LowAmmo** [comment]
@mattgallagher - Got some time to put together the proposed code changes to address the build warning (I at least see the build warning when building with Xcode14).

Also increased the minimum supported iOS version to 12.0 for cocoapods (but, okay sending this back to 9.0, if desired).

Did not modify the cocoapods version - figured that is something to be done when releasing - but can put that on this PR, also.

-Thanks!

**Comment by mattgallagher** [comment]
This should be resolved. 2.1.2 is pushed to Cocoapods. Let me know if you have any further issues.

**Comment by LowAmmo** [comment]
@mattgallagher  - Thanks for merging in the fix (sorry about the formatting being different!)

I'll keep an eye out for when it gets updated in cocoapods!

-Thanks!

**Comment by LowAmmo** [comment]
@mattgallagher  - I reread your comment...and not sure if you know, but 2.1.2 is NOT currently showing up in cocoapods, it's still at 2.1.1.

-Thanks!

**Comment by mattgallagher** [comment]
> @mattgallagher - I reread your comment...and not sure if you know, but 2.1.2 is NOT currently showing up in cocoapods, it's still at 2.1.1.
> 
> -Thanks!

I think the main index just takes a while to update.

I also see 2.1.1 when searching Cocoapods.org but they did send me the "A new version of CwlCatchException was released with version 2.1.2" email (https://github.com/CocoaPods/Specs/commit/826a91e2577d84da5d1cf3a2aab73e3bfdba3a0a) and when I run:

    pod search cwlcatchexception

it shows:

```
-> CwlCatchException (2.1.2)
   A simple Swift wrapper around Objective-C `@try`/`@catch` statements.
   pod 'CwlCatchException', '~> 2.1.2'
   - Homepage: https://github.com/mattgallagher/CwlCatchException
   - Source:   https://github.com/mattgallagher/CwlCatchException.git
   - Versions: 2.1.2, 2.1.1 [trunk repo]
   - Subspecs:
     - CwlCatchException/Tests (2.1.2)

-> CwlCatchExceptionSupport (2.1.2)
   Objective-C internal library used by CwlCatchException
   pod 'CwlCatchExceptionSupport', '~> 2.1.2'
   - Homepage: https://github.com/mattgallagher/CwlCatchException
   - Source:   https://github.com/mattgallagher/CwlCatchException.git
   - Versions: 2.1.2, 2.1.1, 2.1.0 [trunk repo]
```

You might need to run `pod repo update` if these versions don't get used in a `pod update`.

**Comment by LowAmmo** [comment]
@mattgallagher  - Good call!! yeah, I was just checking the website - hadn't run `pod repo update` yet.

Thanks, again!
